README.md

```diff
@@ -434,29 +439,29 @@ ASP.NET Integration
 1. Install the `Sharpbrake.Http.Module` package from NuGet (you can use "Package
    Manager Console" from Visual Studio):
 
-  ```
-  PM> Install-Package Sharpbrake.Http.Module
-  ```
+   ```
+   PM> Install-Package Sharpbrake.Http.Module
+   ```
```

Comment:
Is this change intentional (and other changed related to indentation)? I don't see the reason. This probably doesn't help with the NLog integration.